src/components/10-atoms/carousel/README.md

```diff
@@ -33,7 +33,7 @@ export default AXACarouselReact;
 ```
 
 ```js
-<AXACarouselReact keysenabled="true">
+<AXACarouselReact keysenabled="">
```

Comment:
keysenabled as an empty string? why?

Reply:
true/false here is not correct. i am checking if the attribute is set, and if not, the result will be false. But keysenabled="false" had true as the result.

Reply:
BBut if this only acts as a boolean, you could instead write just `keysenables` without an empty value attached to it.